api/core/workflow/nodes/start/start_node.py

```diff
@@ -24,3 +31,48 @@ def _run(self) -> NodeRunResult:
         outputs = dict(node_inputs)
 
         return NodeRunResult(status=WorkflowNodeExecutionStatus.SUCCEEDED, inputs=node_inputs, outputs=outputs)
+
+    def _validate_and_normalize_json_object_inputs(self, node_inputs: dict[str, Any]) -> None:
+        for variable in self.node_data.variables:
+            if variable.type != VariableEntityType.JSON_OBJECT:
+                continue
+
+            key = variable.variable
+            value = node_inputs.get(key)
+
+            if value is None:
+                if variable.required:
+                    raise ValueError(f"{key} is required in input form")
+                continue
+
+            # 2. 支持前端传字符串或已经是对象
+            if isinstance(value, str):
+                try:
+                    parsed = json.loads(value)
+                except json.JSONDecodeError:
+                    raise ValueError(f"{key} must be a valid JSON object")
+                value = parsed
+
+            if not isinstance(value, dict):
+                raise ValueError(f"{key} must be a JSON object")
+
+            schema = getattr(variable, "json_schema", None)
+
+            if schema:
+                if isinstance(schema, str):
+                    try:
+                        schema_obj = json.loads(schema)
+                    except json.JSONDecodeError:
+                        raise ValueError(f"Invalid JSON schema configured for variable '{key}'")
+                elif isinstance(schema, dict):
+                    schema_obj = schema
+                else:
+                    schema_obj = None
+
+                if schema_obj:
+                    try:
+                        Draft7Validator(schema_obj).validate(value)
```

Comment:
I suggest ensuring the `json_schema`  field is a valid JSON schema before saving it inside workflow DSL.